pyodide-test-runner/pyodide_test_runner/decorator.py

```diff
@@ -16,112 +91,83 @@ def run_in_pyodide(
     packages: Collection[str] = (),
     xfail_browsers: dict[str, str] | None = None,
     driver_timeout: float | None = None,
+    pytest_assert_rewrites: bool = True,
 ) -> Callable:
     """
     This decorator can be called in two ways --- with arguments and without
     arguments. If it is called without arguments, then the `_function` kwarg
-    catches the function the decorator is applied to. Otherwise, standalone
-    and packages are the actual arguments to the decorator.
+    catches the function the decorator is applied to. Otherwise, standalone and
+    packages are the actual arguments to the decorator.
 
     See docs/testing.md for details on how to use this.
 
     Parameters
     ----------
     standalone : bool, default=False
         Whether to use a standalone selenium instance to run the test or not
+
     packages : List[str]
         List of packages to load before running the test
+
+    xfail_browsers: dict[str, str]
+        A dictionary of browsers to xfail the test and reasons for the xfails.
+
     driver_timeout : Optional[float]
-        selenium driver timeout (in seconds)
-    """
+        selenium driver timeout (in seconds). If missing, use the default
+        timeout.
 
+    pytest_assert_rewrites : bool, default = True
+        If True, use pytest assertion rewrites. This gives better error messages
+        when an assertion fails, but requires us to load pytest.
+    """
     xfail_browsers_local = xfail_browsers or {}
 
+    from conftest import ORIGINAL_MODULE_ASTS, REWRITTEN_MODULE_ASTS
```

Comment:
Ideally in the future, we will have to remove circular imports between `conftest.py` and `pyodide-test-runner`. (no action required for this PR)

Reply:
Probably the thing would be to define the pytest hooks in `pyodide_test_runner/pytest_hooks.py` and then import them from there into `conftest.py`.